deepchem/models/torch_models/Text_CNN.py

```diff
@@ -0,0 +1,168 @@
+
```

Comment:
I think this PR doesn't have the utility functions. Can you please double check? You may want to just put up a fresh PR for simplicity